debsources/app/patches/templates/patches/doc_url.html

```diff
@@ -0,0 +1,96 @@
+{#
+  Copyright (C) 2015  The Debsources developers <info@sources.debian.net>.
+  See the AUTHORS file at the top-level directory of this distribution and at
+  https://anonscm.debian.org/gitweb/?p=qa/debsources.git;a=blob;f=AUTHORS;hb=HEAD
+  License: GNU Affero General Public License, version 3 or above.
+#}
+{% extends name+"/base.html" %}
+
+{% block head %}
+{{ super() }}
+<style>
+  .url{display: block; /*margin-left: 50px;*/}
+</style>
+{% endblock %}
+
+{% block title %}URL scheme{% endblock %}
+
+{% block breadcrumbs %}<a href="{{ url_for('.doc') }}">doc</a> / url scheme{% endblock %}
+
+{% block content %}
+{% set url_prefix = "//" + config.SOURCES_URL + '/patches' %}
+
+<h2>{{ self.title() }}</h2>
+
+{% if sources == true %}
+<div class="warning">
+  You can also check the <a href="{{ url_for('sources.doc_url') }}">sources URl scheme</a>
+</div>
+{% endif %}
+{% if copyright == true %}
+<div class="warning">
+  You can also check the <a href="{{ url_for('copyright.doc_url') }}">copyright URl scheme</a>
+</div>
+{% endif %}
+
+<h3>Patches</h3>
+
+<h4>Package summary</h4>
+<p>
+  The URL for the package summary is:
+  <span class="url">{{ url_prefix
+  }}/summary/<strong>packagename</strong>/<strong>version</strong></span>
+  Version can be a suite name, a version number or the keyword 'latest'
+  <a href="{{ url_prefix }}/summary/beignet/1.0.0-1/">See example</a>
+</p>
+
+<h4>Patch view</h4>
+<p>
+  The URL for a path search is:
+  <span class="url">{{ url_prefix
+  }}/patch/<strong>packagename</strong>/<strong>version</strong>/<strong>path</strong></span>
+  Both package name and version are required parameters. The path is the relative path of the patch starting from the debian/patches directory.
+  <a href="{{ url_prefix }}/patch/gnubg/1.02.000-2/debian-changes/">See example</a>
+</p>
```

Comment:
any way to use an example with a patch ending with ".patch", as it's the case for the majority of patches? I'd find this less confusing